execution-environments/community-ee-base/execution-environment.yml

```diff
@@ -3,10 +3,10 @@
 version: 3
 images:
   base_image:
-    name: quay.io/fedora/fedora:latest
+    name: quay.io/fedora/fedora:40
```

Comment:
@anweshadas Why did this change to F40? I remember @Andersson007 pinned F39 in the getting started docs because there was some issue with latest: https://github.com/ansible/ansible-documentation/commit/0de54a9fb271bebd0bec5e0394f5625a7ccb5f27

I guess F39 is EOL now and no longer receiving critical updates so F40 seems like the right choice if we need to pin it. And probably need to update the docs now.

Just want to understand this particular change. Thanks.

Reply:
> I guess F39 is EOL now and no longer receiving critical updates so F40 seems like the right choice if we need to pin it. And probably need to update the docs now.

Yes, since F39 EOL ed yesterday we need to pin it to F40. We need  to update the document.

Reply:
OK. Do we know if F40 has been tested against the EE getting started content?

What was the driver behind the change from `latest` in your PR? That's the main thing I'm curious about. There's no explanation in the PR description and I can't find an issue for that.

Reply:
> OK. Do we know if F40 has been tested against the EE getting started content?
No, we do not have set it as the [release criteria ](https://github.com/ansible-community/images/blob/8cd598cb9a6910d7dad76cc1d147aee81440ce5c/docs/community-ee/community-ee-release-process.md).

Reply:
> What was the driver behind the change from `latest` in your PR? That's the main thing I'm curious about. There's no explanation in the PR description and I can't find an issue for that.

For the containers it is better to pin to an exact version, so that it can be later rebuilt.

Reply:
> > What was the driver behind the change from `latest` in your PR? That's the main thing I'm curious about. There's no explanation in the PR description and I can't find an issue for that.
> 
> For the containers it is better to pin to an exact version, so that it can be later rebuilt.

Sure, I understand the benefit of pinning a version but it isn't clear to me why F40 was chosen. The decision behind that is not visible. Or maybe I'm missing it?

Were there any tests for this or something? Why not pin to F41? Is there a policy of pinning to the lowest supported version maybe?

Reply:
There is no fixed policy as yet. I chose fedora 40 because, 2.17.5-1 had fedora 40 (then latest)
I will pin fedora version to 41 from the next release onward. Also I have opened up an issue to decide on the rule for deciding on feodra version.

Reply:
> For the containers it is better to pin to an exact version, so that it can be later rebuilt.

In that case 40 is a bad choice, since it's not an immutable tag, but something that changes over time.

Reply:
Thanks for pointing that out @felixfontein I've added your comment to [the issue](https://github.com/ansible-community/images/issues/101) since this was already merged.